apps/designer/app/designer/shared/assets/asset-upload.tsx

```diff
@@ -30,29 +30,31 @@ type AssetUploadProps = {
 };
 
 export const AssetUpload = ({ type }: AssetUploadProps) => {
-  const { inputRef, onChange, Form } = useUpload(type);
+  const { inputRef, onChange } = useUpload(type);
   return (
-    <Flex as={Form} css={{ flexGrow: 1 }} onChange={onChange}>
-      <input
-        accept={acceptMap[type]}
-        type="file"
-        name={type}
-        multiple
-        ref={inputRef}
-        style={{ display: "none" }}
-      />
-      <Button
-        variant="blue"
-        type="button"
-        onClick={() => inputRef?.current?.click()}
-        size={2}
-        css={{ flexGrow: 1 }}
-      >
-        <Flex align="center" gap={1}>
-          <UploadIcon />
-          <Text>Upload</Text>
-        </Flex>
-      </Button>
-    </Flex>
+    <form onChange={onChange}>
+      <Flex css={{ flexGrow: 1 }}>
```

Comment:
Will it grow inside form?

Reply:
good catch, it has no need to grow it seems